src/main.cpp

```diff
@@ -425,7 +560,7 @@ bool influxDbWrite() {
 }
 
 void influxDbLoop() {
-  if(v25.size()==0 && wifiOn && cfg.isIfxEnable() && influxDbIsConfigured()){
+  if(v25.size() == 0 && wifiOn && cfg.isApiEnable() && apiIsConfigured() && resetvar != 0 && WrongSerialData == 0){
```

Comment:
same here, I don't know why we need a serial variable validation here, for send the payload to the cloud.

Reply:
done, deleted